### PR TITLE
Tests to ensure the whitelists are up-to-date

### PR DESCRIPTION
**Pull request by pchaigno**
This pull request adds two tests to make sure the hash and project whitelists remain up-to-date.

I also updated submodules during my tests and was able to remove 1 entry from the hash whitelist and 2 from the project whitelist.

### CODE REVIEW

File: vendor/grammars/PHP-Twig.tmbundle [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 9e802d525ed87a90ec7ff842f44c0c992fd56521
+Subproject commit 77def406d70b90dff33d006478198b729e23d22c
```
Comment (arfon) [inline vendor/grammars/PHP-Twig.tmbundle]: Sorry for being a pain but one of the PHP bundles causes test failures in our HTML pipeline tests on GitHub. I'm afraid I've not had a chance to investigate this yet but for now I've been skipping updating`PHP-Twig.tmbundle` and `php.tmbundle` for now. Any chance we can revert these two?

Comment (pchaigno) [inline vendor/grammars/PHP-Twig.tmbundle]: Sure, done.

Comment (arfon) [inline vendor/grammars/PHP-Twig.tmbundle]: 👍 thanks.

**Comment by arfon** [comment]
This looks good to me.

**Pull request merged**